Controllers/param.js

```diff
@@ -0,0 +1,76 @@
+const Cart = require('../Database/cart');
+const User = require('../Database/user');
+const Product = require('../Database/product');
+const Wishlist = require('../Database/wishlist');
+const Address = require('../Database/address');
+var jwt = require('jsonwebtoken');
+
+const isAuthorized = (req, res, next) => {
+	const token = req.headers.authorization;
+	try {
+		const decoded = jwt.verify(token, 'secret');
+		req.user = { userId: decoded.userId };
+		next();
+	} catch (error) {
+		res
+			.status(401)
+			.json({ message: 'Unauthorised access, please add the token' });
+	}
+};
+
+const getCartById = async (req, res, next) => {
+	try {
+		const cart = await Cart.findById(req.user.userId);
```

Comment:
extract `req.user.userId` out and store in a variable for more readability

Reply:
changed!